### PR TITLE
Enhancement/601 subscribe object

### PR DESCRIPTION
**Pull request by DnlLrssn**
**Checklist**

- [ ] tests added
- [x] commits conform to the [Commit message format](./CONTRIBUTING.md#commit)
- [x] documentation updated

**Squash commit message**

* Add subscribeobjects action
* Add unsubscribeobjects action

### CODE REVIEW

File: docs/settingup.md [hunk 1]
```diff
@@ -1165,6 +1165,86 @@ Unpublish sheets in the current app.
 }
 ```
 
+</details><details>
+<summary>subscribeobjects</summary>
```
Comment (carlcervin) [inline docs/settingup.md:1169]: The actions are listed in alphabetical order, so the description of this action needs to be moved.

File: docs/settingup.md [hunk 2]
```diff
@@ -1165,6 +1165,86 @@ Unpublish sheets in the current app.
...
+
+## Subscribeobjects action
+
+Subscribe to any object in the currently active app.
+
+### Settings
+
+* `clear`: Clear any previously subscribed objects from subscription list.
```
Comment (carlcervin) [inline docs/settingup.md:1177]: Suggestion:
"Remove any previously subscribed objects from the subscription list."

File: docs/settingup.md [hunk 3]
```diff
@@ -1165,6 +1165,86 @@ Unpublish sheets in the current app.
...
+* `ids`: List of object ID's to subscribe to.
```
Comment (carlcervin) [inline docs/settingup.md:1178]: Suggestion:
"List of object IDs to subscribe to."

File: docs/settingup.md [hunk 4]
```diff
@@ -1165,6 +1165,86 @@ Unpublish sheets in the current app.
...
+
+### Example
+
+Subscribe to two objects in the currently active app, unsubscribing to any previous subscriptions.
```
Comment (carlcervin) [inline docs/settingup.md:1182]: Suggestion:
"Subscribe to two objects in the currently active app and remove any previous subscriptions."

File: docs/settingup.md [hunk 5]
```diff
@@ -1165,6 +1165,86 @@ Unpublish sheets in the current app.
...
+    "label" : "clear subscriptions and subscribe to mBshXB and f2a50cb3-a7e1-40ac-a015-bc4378773312",
+     "disabled": false,
+    "settings" : {
+        "clear" : true,
+        "ids" : ["mBshXB", "f2a50cb3-a7e1-40ac-a015-bc4378773312"]
+    }
+}
+```
+
+Subscribe to and additional single object (or a list of objects) in the currently active app, adding it to any previous subscriptions.
```
Comment (carlcervin) [inline docs/settingup.md:1196]: Suggestion:
"Subscribe to an additional single object (or a list of objects) in the currently active app, adding the new subscription to any previous subscriptions. "

File: docs/settingup.md [hunk 6]
```diff
@@ -1165,6 +1165,86 @@ Unpublish sheets in the current app.
...
+</details><details>
+<summary>unsubscribeobjects</summary>
+
+## Unsubscribeobjects action
+
+Unsubscribe to any currently subscribed object.
+
+### Settings
+
+* `ids`: List of object ID's to unsubscribe from.
```
Comment (carlcervin) [inline docs/settingup.md:1218]: Suggestion:
"List of object IDs to unsubscribe from."

File: docs/settingup.md [hunk 7]
```diff
@@ -1165,6 +1165,86 @@ Unpublish sheets in the current app.
...
+* `clear`: Clear any previously subscribed objects from subscription list.
```
Comment (carlcervin) [inline docs/settingup.md:1219]: Suggestion:
"Remove any previously subscribed objects from the subscription list."

File: docs/settingup.md (hunk 1 above)
Comment (DnlLrssn) [reply docs/settingup.md:1169]: This file is generated, thought we sorted automatically, maybe it's the order in "groups" which is used will try moving it there and re-generate

Comment (DnlLrssn) [reply docs/settingup.md]: seems to have been it

**Comment by DnlLrssn** [comment]
@carlcervin Think I got all the changes, I re-generated the settingup file and most comments disappeared from "Files changed" list and it was hard to see what they where connected to directly in the PR, tell me if I missed something.

**Comment by SeBBBe** [comment]
This is ready for merging now.

**Comment by DnlLrssn** [comment]
> 
> 
> This is ready for merging now.

as soon as I can make it marked `no conflicts` ...

**Comment by DnlLrssn** [comment]
ok all conflicts should be fixed now

**Pull request merged**